proto/redpanda/api/console/v1alpha1/debug_bundle.proto

```diff
@@ -0,0 +1,54 @@
+syntax = "proto3";
+
+package redpanda.api.console.v1alpha1;
+
+message CreateDebugBundleRequest {
```

Comment:
where does auth come into play?

Reply:
I don't think we need to expose that to the frontend or UX? The backend will set the parameters in the actual Admin API call based on the Admin API settings _within_ console itself.

Reply:
Right now, console must be configured to use a SASL/SCRAM user that is a super user right?  What if in the future it no longer has that power?  Maybe it can be an optional field and if not provided will use console's creds?

I guess for now it's ok, but in the future we may need to include some level of authn options

Reply:
I have added auth parameter in the message.